xbmc/rendering/dx/DeviceResources.h

```diff
@@ -12,9 +12,9 @@
 #include <wrl/client.h>
 #include <concrt.h>
 #if defined(TARGET_WINDOWS_STORE)
-#include <dxgi1_3.h>
+#include <dxgi1_6.h>
 #else
-#include <dxgi1_2.h>
+#include <dxgi1_6.h>
```

Comment:
Why differentiate `TARGET_WINDOWS_STORE` and `TARGET_WINDOWS_DESKTOP`, if include the same file?

Reply:
Yes, now it is redundant. Fixed.